vnext/Desktop.ABITests/ActivationFactory.cpp

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
REVIEW INFO: This file is boilerplate code provided by the C++/WinRT team. It represents a part of the C++/WinRT "runtime" that is currently being worked on, in particular a feature they call "registry-less activation". A variation of this code would likely need to be used in ReactNativeHost.